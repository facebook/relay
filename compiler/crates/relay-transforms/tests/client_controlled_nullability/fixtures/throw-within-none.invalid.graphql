# expected-to-throw
fragment EnabledFoo on User {
  nearest_neighbor @required(action: NONE) {
    name @required(action: THROW)
  }
}
