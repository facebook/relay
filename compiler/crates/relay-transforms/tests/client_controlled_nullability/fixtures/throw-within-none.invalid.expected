==================================== INPUT ====================================
# expected-to-throw
fragment EnabledFoo on User {
  nearest_neighbor @required(action: NONE) {
    name @required(action: THROW)
  }
}
==================================== ERROR ====================================
✖︎ This use of @required is not compatible with Client Controlled Nullability

  throw-within-none.invalid.graphql:3:38
    2 │ fragment EnabledFoo on User {
    3 │   nearest_neighbor @required(action: NONE) {
      │                                      ^^^^
    4 │     name @required(action: THROW)

  ℹ︎ previous state

  throw-within-none.invalid.graphql:4:28
    3 │   nearest_neighbor @required(action: NONE) {
    4 │     name @required(action: THROW)
      │                            ^^^^^
    5 │   }
